classes/02class/exercises/c02-network05/marcio-anatrielo/c02-network05/c02-network05.txt

```diff
@@ -0,0 +1,4 @@
+# AWS VPC + subnets
```

Comment:
file is in the wrong folder. it should be your github username.

Reply:
> need more info on how did you create this igw

First is possible to do this on CLI command as I show the command on exercise file c02-network05.txt and it is possible to do by web interface with the follow steps. 

To create: 
Step 1: Got to Internet Gateways

Step 2: Action

Step 3: Attach VPC

Step 4: Select VPC

Step 5: Deatch from VPC

Step 6: Attach from AWS CLI

To attach: 
aws ec2 attach-internet-gateway --internet-gateway-id igw-0cb74ff15ff9377ee --vpc-id vpc-0c9124eda0e459981

Is that correct?